state/migration_import.go

```diff
@@ -1247,6 +1247,8 @@ func (i *importer) makeApplicationDoc(a description.Application) (*applicationDo
 		MetricCredentials:    a.MetricsCredentials(),
 		DesiredScale:         a.DesiredScale(),
 		Placement:            a.Placement(),
+		// TODO(caas)
```

Comment:
?

Reply:
needs this to land https://github.com/juju/description/pull/76